doc/classes/StringName.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="StringName" version="4.0">
 	<brief_description>
+		A type of string for representing names.
 	</brief_description>
 	<description>
+		[StringName]s are immutable strings designed for general-purpose representing of names. [StringName] ensures that only one instance of given name exists (so two [StringName]s with the same value are the same object). Comparing them is much faster than with regular [String]s, because only the pointers are compared, not the whole strings.
```

Comment:
```suggestion
		[StringName]s are immutable strings designed for general-purpose represention of unique names. [StringName] ensures that only one instance of a given name exists (so two [StringName]s with the same value are the same object). Comparing them is much faster than with regular [String]s, because only the pointers are compared, not the whole strings.
```